Userland/Applications/Browser/InspectorWidget.cpp

```diff
@@ -91,6 +92,11 @@ InspectorWidget::InspectorWidget()
         set_selection(index);
     };
 
+    auto& accessibility_tree_container = top_tab_widget.add_tab<GUI::Widget>("Acessibility");
```

Comment:
Spell check accessibility. Also some fixes pending in the commit message (thee and descriptin).